bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogger.java

```diff
@@ -802,88 +876,217 @@ private long readLastLogId(File f) {
         }
     }
 
+    interface EntryLogManager {
```

Comment:
Move into its own java file. EntryLogger is already huge.

Reply:
will do